src/Form/Form.tsx

```diff
@@ -1,9 +1,56 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./Form.css";
 
-const Form = () => {
+export interface Quote {
+  q: string;
+  a: string;
+  h: string;
+}
+
+interface formProps {
+  setAllQuotes: React.Dispatch<React.SetStateAction<Quote[]>>
+}
+
+
+interface Mood{
+  k: string;
+  l: string;
+}
```

Comment:
Let's use this an an opportunity to practice speaking/writing about this new tech. Anyone care to explain this code to me?

Reply:
An interface is how we define reusable variables, props, functions, etc in TypeScript. For example, the export interface on line 4 is defining our Quote that we will receive from our API. We're defining each property of the Quote object as the data type that we will receive. In this case, "q", "a" and "h" are all strings. So, if we tried to pass "q" as a boolean, TypeScript would throw an error and tell us that the data type of boolean cannot be used for that attribute. In the most recent PR, we updated all of our components so that they import a `QuoteInterface` from an `interfaces.tsx` file.